src/v2/Components/Cells/CellSale.story.tsx

```diff
@@ -0,0 +1,54 @@
+import { graphql } from "relay-runtime"
+import { SystemQueryRenderer } from "v2/System/Relay/SystemQueryRenderer"
+import { CellSaleStoryQuery } from "v2/__generated__/CellSaleStoryQuery.graphql"
+import { CellSaleFragmentContainer, CellSalePlaceholder } from "./CellSale"
+
+export default {
+  title: "Components/Cell",
+}
+
+export const CellSale = () => {
+  return (
+    <SystemQueryRenderer<CellSaleStoryQuery>
+      placeholder={<CellSalePlaceholder />}
+      query={graphql`
+        query CellSaleStoryQuery {
+          viewer {
+            salesConnection(
+              first: 99
+              published: true
+              live: true
+              sort: LICENSED_TIMELY_AT_NAME_DESC
+            ) {
+              edges {
+                node {
+                  ...CellSale_sale
+                }
+              }
+            }
+          }
+        }
+      `}
+      render={({ error, props }) => {
+        if (error) {
+          console.error(error)
+          return null
+        }
+
+        if (!props?.viewer?.salesConnection) {
+          return <CellSalePlaceholder />
+        }
+
+        return (
+          <CellSaleFragmentContainer
+            sale={props.viewer?.salesConnection?.edges?.[0]?.node!}
```

Comment:
Sales are published and unpublished all the time, so rather than fix things to an id (brittle), use a connection and grab the first item

Reply:
**suggestion (non-blocking):** A comment might be useful here; it may be difficult or impossible to find this PR comment if that context is handy. 🤷

Reply:
It's a storybook story, so not too worried!